File: starters/java/scrabble/Scrabble.java
// TODO: 实现 Scrabble.java
import java.util.Scanner;

public class Scrabble {
    // Scrabble 字母分值
    private static final int[] POINTS = {
        1, 3, 3, 2, 1, 4, 2, 4, 1, 8, 5, 1, 3,
        1, 1, 3, 10, 1, 1, 1, 1, 4, 4, 8, 4, 10
    };

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // 获取两位玩家的单词
        System.out.print("Player 1: ");
        String word1 = scanner.nextLine();

        System.out.print("Player 2: ");
        String word2 = scanner.nextLine();

        // 计算每个单词的分数
        int score1 = computeScore(word1);
        int score2 = computeScore(word2);

        // 输出获胜者
        if (score1 > score2) {
            System.out.println("Player 1 wins!");
        } else if (score2 > score1) {
            System.out.println("Player 2 wins!");
        } else {
            System.out.println("Tie!");
        }

        scanner.close();
    }

    // 计算单词的 Scrabble 分数
    private static int computeScore(String word) {
        // TODO: 实现分数计算
        return 0;
    }
}

File: starters/python/scrabble/scrabble.py
# TODO: 实现 scrabble.py

# Scrabble 字母分值 (A=1, B=3, C=3, ...)
POINTS = [1, 3, 3, 2, 1, 4, 2, 4, 1, 8, 5, 1, 3, 1, 1, 3, 10, 1, 1, 1, 1, 4, 4, 8, 4, 10]


def compute_score(word):
    """计算单词的 Scrabble 分数"""
    # TODO: 实现分数计算
    return 0


def main():
    # 获取两位玩家的单词
    word1 = input("Player 1: ")
    word2 = input("Player 2: ")

    # 计算每个单词的分数
    score1 = compute_score(word1)
    score2 = compute_score(word2)

    # 输出获胜者
    if score1 > score2:
        print("Player 1 wins!")
    elif score2 > score1:
        print("Player 2 wins!")
    else:
        print("Tie!")


if __name__ == "__main__":
    main()
